### PR TITLE
Admin verb to re-enable the gravgen.

### PR DESCRIPTION
**Pull request by FlavoredCactus**
-admin message when someone shuts off the GravGen
-Fixs a backpack's incorrect name
-Asay will now always be enabled despite admin messages being turned
off. deadmin if you dont wish to see asay

### CODE REVIEW

**Comment by xXTheDracheXx** [comment]
Alright here's my feedback on the verb...

1. It should probably be in the secrets panel.
2. It should be able to toggle it on and off.
3. You should be able to choose on which z-level it turns the gravity gen on/off.
4. Rename it to something more clear. "fix_gravity" isn't clear. Something like "toggle_gravity_gen" would be much better.
5. Description is probably not needed, if it is, it should be professional.

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -828,6 +828,23 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	message_admins("[key_name(src)] cleaned air within [size] tiles.")
 	log_game("[key_name(src)] cleaned air within [size] tiles.")
 
+/client/proc/fix_gravity()
+	set name = "Turn on gravity"
+	set category = "Special Verbs"
+	set desc = "Use this when griefons turn off the gravity gen "
+	for(var/obj/machinery/gravity_generator/main/grav in world)
+		if(grav.z == 1) // make sure the grav gen is in z 1
```
Comment (xXTheDracheXx) [inline code/modules/admin/verbs/randomverbs.dm]: 6. Unnecessary comments. It's pretty clear what it does.

File: code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -828,6 +828,23 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
 
+/client/proc/fix_gravity()
+	set name = "Turn on gravity"
+	set category = "Special Verbs"
+	set desc = "Use this when griefons turn off the gravity gen "
+	for(var/obj/machinery/gravity_generator/main/grav in world)
+		if(grav.z == 1) // make sure the grav gen is in z 1
+			grav.breaker = 1
+			grav.charge_count = 100
+			grav.set_state(1) // Turn it on!!
```
Comment (xXTheDracheXx) [inline code/modules/admin/verbs/randomverbs.dm]: I haven't looked in the gravity generator code but I'd be willing to bet that there's atleast an update_icon function which would do the job of updating the overlays for you.

Comment (FlavoredCactus) [inline code/modules/admin/verbs/randomverbs.dm]: nope, doesnt do that.

**Comment by FlavoredCactus** [comment]
UPDATED

1) Putting it in secrets would make it primary+ its fine where it is
2) Its not ment to be an event button, its ment to be a fix
3) ^same as above
4) done
5) done

**Comment by xXTheDracheXx** [comment]
1. You can probably make a tab for admins or less.
2. Regardless of what it is meant to be, I don't understand why you wouldn't make it a toggle.
3. Meh.

**Pull request merged**